analyzers/tests/SonarAnalyzer.UnitTest/TestCases/MemberOverrideCallsBaseMember.CSharp10.Fixed.cs

```diff
@@ -34,3 +34,32 @@ record Derived : Base
     public override int MyProperty13 { get; init => base.MyProperty13 = value; } // Error [CS0501] 'member function' must declare a body because it is not marked abstract, extern, or partial
     public override int MyProperty14 { get => base.MyProperty14; init => base.MyProperty14 = value; } // Error [CS0115, CS0117, CS0117] no suitable method found to override, 'Base' does not contain a definition for 'MyProperty14'
 }
+
+namespace CompilerGeneratedMethods
+{
+    record BaseWithOverride
+    {
+        public override string ToString() => "Some custom ToString";
+    }
+
+    record Derived : Base
```

Comment:
does it make sense to also add test cases with `Equals` and `GetHashCode` for records?

what about `!=` and `==`?

Reply:
I checked the generated code for Equals and GetHashCode. Both are generating some code and call base. It might be worth to also include them as there are use cases were you want to exclude your members from equality. Then you just call base without other logic.
The operators don't have base calls (are static).